be/src/exec/json_scanner.cpp

```diff
@@ -843,4 +854,14 @@ Status JsonReader::_construct_column(simdjson::ondemand::value& value, Column* c
     return add_adaptive_nullable_column(column, type_desc, col_name, &value, !_strict_mode);
 }
 
+void JsonReader::_append_error_msg(std::string&& row, const std::string& error_msg) {
```

Comment:
It is a little uncommon to use right value reference as the input parameter here. consider using string_view or std::string const reference?

Reply:
use `const std::string&` instead